packages/suite/src/components/wallet/CoinjoinSummary/CoinjoinStatus.tsx

```diff
@@ -279,18 +292,22 @@ export const CoinjoinStatus = ({ session, accountKey }: CoinjoinStatusProps) =>
             );
         }
 
-        if (phase !== undefined) {
+        if (sessionPhase !== undefined) {
             return (
-                <>
-                    <Translation id={COINJOIN_PHASE_MESSAGES[phase]} />
-                    <p>
-                        <CountdownTimer
-                            isApproximate
-                            deadline={phaseDeadline}
-                            format={getPhaseTimerFormat(phaseDeadline)}
-                        />
-                    </p>
-                </>
+                <TextCointainer>
+                    <PhaseMessage isErrorPhase={getIsErrorSessionPhase(sessionPhase)}>
+                        <Translation id={SESSION_PHASE_MESSAGES[sessionPhase]} />
+                    </PhaseMessage>
+                    {roundPhaseDeadline && (
+                        <p>
```

Comment:
See console: `Warning: validateDOMNesting(...): <p> cannot appear as a descendant of <p>`